src/WebJobs.Extensions.DurableTask.Analyzers/Analyzers/Orchestrator/ThreadTaskAnalyzer.cs

```diff
@@ -107,7 +107,7 @@ private static bool AnalyzeIdentifierThread(SyntaxNode method, CompilationAnalys
                         {
                             if (memberSymbol != null && memberSymbol.ToString().StartsWith("System.Threading.Thread"))
                             {
-                                var diagnostic = Diagnostic.Create(Rule, memberAccessExpression.GetLocation(), memberAccessExpression);
+                                var diagnostic = Diagnostic.Create(Rule, memberAccessExpression.GetLocation(), "Thread.Start");
```

Comment:
Is there a reason this changes from `memberAccessExpression`?

Reply:
This is related to "Removed some of the custom messages." Some of these messages get way too long. The memberAccessExpression still gets underlined, but I didn't think it was necessary to put everything that was before the .Start or .ContinueWith in the custom message. Customers can use something like this:

`context.CallActivityWithRetryAsync("HelloWorld", retryOptions, "Hello").ContinueWith(i =>
                        context.SetCustomStatus(
                            $"Retrieved name"));`
                            
The messages get too long quickly and it becomes even less clear what the problem is.